app/controllers/applicants_controller.rb

```diff
@@ -0,0 +1,29 @@
+class ApplicantsController < ApplicationController
+  def new
+  end
+
+  def show
+    @applicant = Applicant.find(params[:id])
+  end
+
+  def create
```

Comment:
Looks good, for refactoring we should consider making a private method to pass the applicant params through.